assets/js/ajax-filters.js

```diff
@@ -255,26 +262,67 @@ jQuery( document ).ready( function ( $ ) {
 		$supports_html5_history = false;
 	}
 
-	var location = document.location.href.split('#')[0];
-
 	function job_manager_store_state( target, page ) {
+		// changed this to ? because this is the standard query string identifier in php
+		// and changed to window.location for better cross browser support
+		// and moved into the function so it rechecks everytime the function is called
+		var location = window.location.href.split('?')[0];
+		var query = window.location.href.split('?')[1];
+		query = query.split('&');
```

Comment:
Would rather not add more libs if possible - I'd stick with the hash which avoids the issue completely.